test/integration/integration_test.go

```diff
@@ -43,7 +45,7 @@ const (
 	// scripts are made available for integration testing.
 	//
 	// Port 23456 must match the port used in the fake image Dockerfiles
-	FakeScriptsHttpURL = "http://127.0.0.1:23456/sti-fake/.sti/bin"
+	FakeScriptsHTTPURL = "http://127.0.0.1:23456/sti-fake/.sti/bin"
```

Comment:
OK ... I had changed it as a result of golint output.  If it is everyone's preference to buck the golint recommendation, I can switch it.